learn_ai/assignments/rag_assignment_2.ipynb

```diff
@@ -0,0 +1,1544 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    from fastembed import TextEmbedding
</pre><p>Thanks for using FastEmbed!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/FuturePathAI/Learn-AI-Engineering/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>